include/alpaka/atomic/AtomicAtomicRef.hpp

```diff
@@ -0,0 +1,231 @@
+/* Copyright 2021 Felice Pantaleo (CERN), Andrea Bocci (CERN)
```

Comment:
No need for the affiliation here. We will correctly reference it in the 0.9 release which will be a citable publication with you as co-authors.